modules/planning/tasks/deciders/path_reuse_decider/path_reuse_decider.cc

```diff
@@ -34,6 +35,7 @@ using apollo::common::math::Vec2d;
 
 int PathReuseDecider::reusable_path_counter_ = 0;
 int PathReuseDecider::total_path_counter_ = 0;
+bool PathReuseDecider::path_reusable_ = false;
```

Comment:
You may init value inline in the header file.

Reply:
Can you please give me an example?

Reply:
Oops, not work for non-const static member.